ydb/core/ydb_convert/ut/ya.make

```diff
@@ -14,7 +14,7 @@ SRCS(
 
 PEERDIR(
     library/cpp/testing/unittest
-    ydb/core/testlib/default
+    ydb/core/testlib/pg
```

Comment:
is it intentional?

Reply:
Yes, testlib/default has dummy implementation for pg types interfaces and catalog